src/controller/api.js

```diff
@@ -0,0 +1,42 @@
+/* eslint-disable comma-dangle */
+const API = (() => {
+  const getDataByIpCheck = async () => {
+    const response = await fetch(
+      `http://api.ipstack.com/check?access_key=${process.env.API_KEY_IPSTACK}&format=1`
+    );
+    return response.json();
+  };
+
+  const weatherDataByCity = async (cityName) => {
+    const response = await fetch(
+      `http://api.openweathermap.org/data/2.5/weather?q=${cityName}&appid=${process.env.API_KEY_OPENWEATHERMAP}&units=metric`
+    );
+    return response.json();
+  };
+
+  const sanitizeNames = (name) => name.trim().toLowerCase().replace(/\s/g, '-');
+
+  const getDefaultImage = () => process.env.DEFAULT_IMAGE;
+
+  const cityBackgroundImage = async (cityName) => {
+    try {
+      const response = await fetch(
+        `https://api.teleport.org/api/urban_areas/slug:${sanitizeNames(
+          cityName
+        )}/images/`
+      );
+      const resp = await response.json();
+      return resp.photos[0].image.web;
+    } catch (error) {
+      return getDefaultImage();
+    }
+  };
```

Comment:
## Get default image

If not find the city it returns the default city. It's inside try-catch block

Reply:
Yes, you have done well.
Now, let's add a div to display to the user if the city is not found, something like `city not found`, let's try to tell the user his/her word is bogus 😄 
for some reason, I couldn't start the app locally, could that be from my end 🤔 
example; if the user should type `new yor` nothing happens on the app until you check the console.

[Optional]
Note: 
we could search for a country name, some are having the correct default image you added while some are wrong.
I am still trying to figure it out why 🤔 

Correct!
![image](https://user-images.githubusercontent.com/31897434/90573200-c346a480-e1ad-11ea-8dbd-85546143bce9.png)

Wrong!
![image](https://user-images.githubusercontent.com/31897434/90573373-3c45fc00-e1ae-11ea-9c00-888c27521407.png)